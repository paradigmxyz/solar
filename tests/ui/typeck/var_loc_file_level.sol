struct S {
    uint x;
}

uint memory constant a0 = 0;    //~ ERROR: data locations are not allowed here
uint[] memory constant b0 = []; //~ ERROR: data locations are not allowed here
S memory constant c0 = S(0);    //~ ERROR: data locations are not allowed here
S[] memory constant d0 = [];    //~ ERROR: data locations are not allowed here
