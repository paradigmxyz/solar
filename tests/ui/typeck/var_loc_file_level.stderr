error: data locations are not allowed here
  --> ROOT/tests/ui/typeck/var_loc_file_level.sol:LL:CC
   |
LL | uint memory constant a0 = 0;
   |      ^^^^^^
   |

error: data locations are not allowed here
  --> ROOT/tests/ui/typeck/var_loc_file_level.sol:LL:CC
   |
LL | uint[] memory constant b0 = [];
   |        ^^^^^^
   |

error: data locations are not allowed here
  --> ROOT/tests/ui/typeck/var_loc_file_level.sol:LL:CC
   |
LL | S memory constant c0 = S(0);
   |   ^^^^^^
   |

error: data locations are not allowed here
  --> ROOT/tests/ui/typeck/var_loc_file_level.sol:LL:CC
   |
LL | S[] memory constant d0 = [];
   |     ^^^^^^
   |

error: aborting due to 4 previous errors

